lib/api/createFile.js

```diff
@@ -114,7 +114,7 @@ module.exports = function createFile(bucketId, fileName, file, opts, cb) {
 // isn't one already
 function fileToStream(file) {
   // Is it already a readable stream?
-  if(file instanceof stream.Readable) {
+  if(file.readable) {
```

Comment:
is-stream just adds these two checks to the current check for readable property

 `typeof stream._read === 'function' && typeof stream._readableState === 'object'`

so I suggest instead of requiring that module we just do this.

```
  if(file.readable && typeof file._read === 'function' && typeof file._readableState === 'object') {
    console.log('wee')
    return file;
  }
```